Automattic/jetpack
# Open Graph: Always output tags if they haven't been.

Currently we only output the Open Graph tags if the sharing or Publicize modules are enabled and there are no known conflicting plugins active.

## Proposed changes:

This removes the module check, but adds a check where we capture the output of wp_head, and then see if any of the tags have been output already. If not then we render them.

This change means that the default behaviour is flipped, so that Open Graph tags are always rendered, unless they're disabled.

I'm also slightly concerned about using output buffering, as I don't know if that will affect performance adversely, or if it could cause other problems.

Fixes #39225

### Other information:


## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Check that OG tags are rendered when using a block theme
* Yoast already disables Jetpack OG rendering. There might be another plugin we can use for testing.
* Add the web-stories plugin https://wordpress.org/plugins/web-stories/ , create a story and check that Open Graph tags are rendered
